src/test/scala/pirate.example/GitExample.scala

```diff
@@ -1,12 +1,14 @@
 package pirate.example
 
 import pirate._, Pirate._
-import scalaz._, effect.IO
+import scalaz._, Scalaz._, effect.IO
+
+import java.io.File
 
 case class Git(
-  cwd: String,
-  conf: String, // FIX support a read like thing for x=y
-  exec: String,
+  cwd: Option[String],
```

Comment:
Any reason for these being `Option` it can lead to awkward usage, it is not a big deal, but if there is a reason why making them Option is easier than threading the defaults through I would like to understand what prevents it.